vm/systemSmartContracts/esdt.go

```diff
@@ -1345,6 +1404,15 @@ func (e *esdt) transferNFTCreateRole(args *vmcommon.ContractCallInput) vmcommon.
 		return vmcommon.FunctionWrongSignature
 	}
 
+	if token.CanCreateMultiShard {
+		lastIndex := len(args.CallerAddr) - 1
+		isLastByteTheSame := args.Arguments[1][lastIndex] == args.Arguments[2][lastIndex]
+		if !isLastByteTheSame {
+			e.eei.AddReturnMessage("transfer NFT create can happen for addresses which end in the same byte")
```

Comment:
as this will be an error message, please rephrase:
`transfer NFT create cannot happen for addresses that don't end in the same byte`

Reply:
done